kamailio/presence-fast-pickup.cfg

```diff
@@ -88,6 +88,7 @@ route[PRESENCE_FAST_PICKUP_ATTEMPT]
                   }
                   remove_hf_re("^Replaces");
                   append_hf("Replaces: $var(rep)\r\n");
+                  append_hf("k-cid: $var(replaced_call_id)\r\n");
```

Comment:
@lazedo custom SIP headers should start with X-, even if its internal we should keep with the standard ;)

Reply:
this is between kamailio and fs and serves only for packet capture correlation. we don't want to include X-* as it will end up in kazoo which is not intended, this also doesn't pass thru route_req a direct replace is made in freeswitch